scripts/retry.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+set -euo pipefail
+
+max_retries=${MAX_RETRIES:-7}
+backoff=${BACKOFF:-1}
+
+retries=0
+until (( retries == max_retries )) || "${@}"; do
+    sleep "$(( (retries++)*backoff ))"
```

Comment:
very nice, here we have backoff 👏 see https://github.com/mongodb/mongodb-atlas-kubernetes/pull/1519#discussion_r1570038910

Reply:
i'd like to see jitter as well here but backoff is good enough.